gnome/gobject-introspection/files/patch-fix-scanner-in-build-execution.diff

```diff
@@ -0,0 +1,61 @@
+--- giscanner/ccompiler.py.orig
++++ giscanner/ccompiler.py
+@@ -184,7 +184,10 @@
+         if os.name == 'nt':
+             runtime_path_envvar = ['LIB', 'PATH']
+         else:
+-            runtime_path_envvar = ['LD_LIBRARY_PATH', 'DYLD_FALLBACK_LIBRARY_PATH']
++            if sys.platform == "darwin":
++                runtime_path_envvar = ['DYLD_LIBRARY_PATH']
++            else:
++                runtime_path_envvar = ['LD_LIBRARY_PATH']
+             # Search the current directory first
+             # (This flag is not supported nor needed for Visual C++)
+             args.append('-L.')
+--- giscanner/dumper.py.orig
++++ giscanner/dumper.py
+@@ -236,8 +236,25 @@
+ 
+         args.extend(sources)
+ 
+-        pkg_config_libs = pkgconfig.libs(self._packages,
++        pkg_config_libs, pkg_config_libs_only_L = pkgconfig.libs(self._packages,
```

Comment:
Before submitting this upstream, you'll likely want to fix the indentation, either by moving the `msvc_syntax` param in the next line to the same column as the first argument of the call to `pkgconfig.libs()`, or, since the line is getting long anyway, by moving `self._packages` indented to the next line.

Reply:
Here's my proposal on submitting upstream:

Given how far our GO-I is behind current release / master, I'm not going to take the time to work out -if- this is even an issue with the current release / GIT master.

-When- we update GO-I to a meson-based release -- hopefully the current GO-I release -- then I'll work through this issue again determine whether any changes are needed, and if so then update the patches here in MP -- as well as submit them upstream.

Reply:
Agreed on the reformatting. Given that this patch is a one-off & not to be submitted upstream (IMHO at least), I'm OK with the formatting.